xdsl/utils/lexer.py

```diff
@@ -215,6 +215,23 @@ class Kind(Enum):
 
     span: Span
 
+    @property
+    def text(self):
```

Comment:
no need for `get_text` I guess

Reply:
In that case, I would argue that this is a simple enough operation that we can consider it a property.
`get_text` would mean that this has somewhat of a cost I think.